share/mk/src.opts.mk

```diff
@@ -432,10 +432,8 @@ BROKEN_OPTIONS+= TESTS
 .if ${.MAKE.OS} != "FreeBSD"
```

Comment:
This whole .if is a local diff in CheriBSD that upstream doesn't seem to need, so we can probably drop it?

Reply:
Yes. Probably it got upstreamed somewhere slightly different so its eventual removal didn't result in this block being removed.